### PR TITLE
Bump slack-notifier Gem Dependency (take 2)

### PR DESCRIPTION
**Pull request by joemasilotti**
This PR is a fix for #11446.

### CODE REVIEW

File: fastlane/lib/fastlane/actions/slack.rb [hunk 1]
```diff
@@ -22,29 +22,34 @@ def self.run(options)
...
 
-        notifier.username = options[:use_webhook_configured_username_and_icon] ? nil : options[:username]
-        icon_url = options[:use_webhook_configured_username_and_icon] ? nil : options[:icon_url]
+        username = options[:use_webhook_configured_username_and_icon] ? nil : options[:username]
 
-        if options[:channel].to_s.length > 0
-          notifier.channel = options[:channel]
-          notifier.channel = ('#' + notifier.channel) unless ['#', '@'].include?(notifier.channel[0]) # send message to channel by default
+        notifier = Slack::Notifier.new(options[:slack_url]) do
+          defaults channel: channel,
```
Comment (KrauseFx) [inline fastlane/lib/fastlane/actions/slack.rb]: This is a method call, right? if so, can we surround it with `( )`?

File: fastlane/lib/fastlane/actions/slack.rb [hunk 2]
```diff
@@ -22,29 +22,34 @@ def self.run(options)
...
+
         slack_attachment = generate_slack_attachments(options)
 
         return [notifier, slack_attachment] if Helper.is_test? # tests will verify the slack attachments and other properties
 
         begin
-          result = notifier.ping '',
-                                 icon_url: icon_url,
-                                 attachments: [slack_attachment]
+          results = notifier.ping '',
```
Comment (KrauseFx) [inline fastlane/lib/fastlane/actions/slack.rb]: Same here https://github.com/fastlane/fastlane/pull/11465/files#r160184286

**Review by KrauseFx — APPROVED** [review]
Looks solid, thanks for the update. One small change before it's ready to be merged 👍

**Comment by joemasilotti** [comment]
Done.

**Comment by joemasilotti** [comment]
Sorry to be a pain, but I don't understand the issue that CI brought up. It seems to be mentioning something about documentation, but I didn't change any. Please advise.

**Comment by KrauseFx** [comment]
Ah, don't worry about Ubuntu CI. Can you resolve the merge conflicts, then we can do another review 👍 Thanks and sorry it took so long

**Comment by janpio** [comment]
(The Ubuntu failure is a bit strange, I am investigating it. Solution to it would be to rebase the branch onto current master - then the docs tests should succeed as well.)

**Comment by joemasilotti** [comment]
Rebased and pushed!

**Comment by janpio** [comment]
Sorry you are having such a rough time with this PR, there were some style checks added. `bundle exec rubocop -a` will probably automatically fix them, what it wants is for you to add `( ... )` around these method params. Probably got lost during the rebase or kept because you edited stuff in these lines.

**Comment by joemasilotti** [comment]
Ah, yeah, I definitely lost them in the rebase. I didn't see the parens wrapping the gems. The last commit should take care of it.

**Comment by janpio** [comment]
Wheeee! 🥇

**Comment by KrauseFx** [comment]
Yesss, awesome, thanks you two 👍

**Pull request merged**